README.md

```diff
@@ -1,96 +1,230 @@
 # kube-aliases
 
-This is an oh-my-zsh plugin (or source `kube-aliases.plugin.zsh` for bash) to
-make working with Kubernetes easier. It provides a bunch of bash aliases and
-Zsh functions. 
+kube-aliases comes with a comprehensive set of aliases for Kubernetes. If
+desired, there is a utility tool to generate custom aliases or to change the
+aliases automatically. For example, if you prefer `rm` to `del` for deleting.
+
+- [kube-aliases](#kube-aliases)
+- [Usage](#usage)
+  * [General Alias Rules](#general-alias-rules)
+  * [Generating Aliases](#generating-aliases)
+    + [Customizing Aliases Yaml](#customizing-aliases-yaml)
+      - [resources](#resources)
+      - [cmds](#cmds)
+      - [additional](#additional)
+- [Installation](#installation)
+  * [Bash Example](#bash-example)
+  * [Zsh Example](#zsh-example)
+  * [Generate Aliases](#generate-aliases)
+  * [AUR](#aur)
+ 
+    
+# Usage
+
+## General Alias Rules
+
+Aliases follow the following conventions:
 
-## Usage
+```bash
+k           # kubectl
+kc<r>       # kubectl create <resource>, e.g. kcd for kubectl create deployment
+kdel<r>     # kubectl delete <resource>, e.g. kdelp for kubectl delete pods
+kd<r>       # kubectl describe <resource>, e.g. kdp for kubectl describe pod
+ke<r>       # kubectl edit <resource>, e.g. kep for kubectl edit pods
+kg<r>       # kubectl get <resource>, e.g. kgp for kubectl get pods
+kga<r>      # kubectl get --all-namespaces -o wide <resource>, e.g. kgap for kubectl --all-namespaces -o wide get pods
+```
+
+## Generating Aliases
+
+Want to generate custom aliases? [Install](#generate-aliases) the latest `generate-kube-aliases`
```

Comment:
I feel like this should link to the releases page

Reply:
I'm planning on adding an Arch Linux package that they could install or if we wanted to add a brew install. Also was planning on adding a regular old curl. Not sure what you would prefer if we add those sections. Link to release or the installation options.

Reply:
Probably both